components/Composition/FAnnouncementCard.jsx

```diff
@@ -0,0 +1,125 @@
+import { StyleSheet, View } from 'react-native';
+import { FImage } from 'components/Composition/FImage';
+import sizes from 'themes/sizes';
+import fonts from 'themes/fonts';
+import { FHeading } from 'components/Composition/FHeading';
+import React from 'react';
+import { TouchableWithoutFeedback } from 'react-native-web';
+import { FHeadingWithIcon } from 'components/Composition/FHeadingWithIcon';
+import icons from 'themes/icons';
+import colors from 'themes/colors';
+import { FCard } from 'components/Composition/FCard';
+import placements from 'themes/placements';
+import { useNavigation } from '@react-navigation/native';
+import PropTypes from 'prop-types';
+
+export const FAnnouncementCard = ({
+  width, data, height, link, titleStyle,
+}) => {
+  const navigation = useNavigation();
+  return (
+    <TouchableWithoutFeedback onPress={() => navigation.push(link)}>
+      <View style={{
+        padding: sizes.PADDING_5,
+      }}
+      >
+        <FCard
+          width={width}
+          paddingHorizontal={sizes.PADDING_10}
+          paddingVertical={sizes.PADDING_10}
+        >
+          <View style={{
+            minHeight: height,
+            maxHeight: height,
+          }}
+          >
+            <FImage
+              imagePath=""
```

Comment:
FImage ma jeszcze imageWidth który trzeba mu ustawić na 100%, to samo z imageHeight